CONTRIBUTING.rst

```diff
@@ -63,7 +63,18 @@ Conda quickly installs, runs and updates packages and their dependencies.
 It also allows to easily create, save, load and switch between different
 environments on your local computer.
 
-The developer environment can be created using the
+Before continuing, Mac OSX users also need to install a more recent compiler.
+See instructions here: :ref:`OSX users <install_osx_users>`.
+
+The developer environment can be installed in two different ways. In case of installation
+or import errors, try both approaches. The first approach starts by creating an empty
+python environment::
+
+    conda create -n pysteps_dev python=3.8
+
+Pip will take care to install the necessary dependencies in this environment (see below).
```

Comment:
Although this solves the reported issue, mixing pip and conda may cause issues in the future ([more info here](https://www.anaconda.com/blog/using-pip-in-a-conda-environment)).  For example, when a package is updated using `conda update <a packages>` and a package installed by pip is a dependency.

Perhaps this can be described as a second option in the case that using `environment_dev.yml` fails.

Reply:
I agree, it is a good idea to mention this only as the second option. In fact, starting from an empty conda env followed by pip will increase the chances of having to install other conda packages later. The article you sent says to install pip packages only after having installed all the conda ones.

Reply:
I agree, it is a good idea to mention this only as the second option. In fact, starting from an empty conda env followed by pip will increase the chances of having to install other conda packages later. The article you sent says to install pip packages only after having installed all the conda ones.